erc20-price/src/lib.rs

```diff
@@ -188,3 +190,157 @@ fn store_chainlink_price(
         }
     }
 }
+
+#[substreams::handlers::store]
+fn store_pair_created_events(block: eth::Block, output: StoreSetProto<PairCreatedEvent>) {
+    for log in block.logs() {
+        if let Some(event) = factory::events::PairCreated::match_and_decode(log) {
```

Comment:
I wonder if this catches ALL uniswap forks instead of just uniswap. Can you investigate?

Reply:
yes, this works for all the Uniswap forks.

Reply:
since this will be used by a lot of people, I'd rather be selective about which price feed to offer. I think the filtering should be happening inside of this module instead of outside

Reply:
yeah, that makes sense. I will add a check to filter out prices only from known factory addresses.